gossipsub.go

```diff
@@ -1909,6 +1917,10 @@ func (gs *GossipSubRouter) getPeers(topic string, count int, filter func(peer.ID
 	return peers
 }
 
+func (gs *GossipSubRouter) WithTagTracerPubsubOption() Option {
```

Comment:
document this please!
Also, lets call it `WithDefaultTagTracer`.

Reply:
https://github.com/libp2p/go-libp2p-pubsub/pull/503/commits/03b0d805018ea5e406a157eda8d708cce8b65b14